.travis.yml

```diff
@@ -9,6 +9,7 @@ node_js:
   - "0.10"
 env:
   - TEST_DIR=packages/fluxible
+  - TEST_DIR=packages/fluxible-reducer-store
```

Comment:
It creates a build matrix that has a job for each combination of package and node version. Each TEST_DIR will run the CI on only that directory. See the results here: https://travis-ci.org/yahoo/fluxible/builds/84579529